README.md

```diff
@@ -39,11 +39,13 @@ This document attempts to codify that knowledge for people to (re)discover the a
 
 * Parallelize requests to different services
 * Request hedging (send redundant requests to multiple replicase, use response from fastest one)
+* Use SIMD instructions when possible
```

Comment:
"When possible" here should probably have an additional conditional qualification based on the input data. See e.g. [this analysis of simdjson](https://github.com/bytedance/sonic/blob/main/INTRODUCTION.md#why-is-simdjson-go-not-fast-enough).

Reply:
Taking better advantage of SIMD and yielding better performance is problem-specific and depends on the compiler and the user optimizations.
Maybe better a statement would be: Use optimized SIMD instructions for suitable problems?